cow_amm/cow_amm_lp_and_tvl_4047078

```diff
@@ -0,0 +1,48 @@
+with 
```

Comment:
Can you comment the high level query file (documenting required parameters) and even the subqueries (I think the other files in the repo can serve as an example) so that it's clear what and how this query is doing things?